core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -493,58 +494,45 @@ private static void checkWorkerPorts() {
   }
 
   /**
-   * {@link PropertyKey#USER_FILE_BUFFER_BYTES} should not bigger than {@link Integer#MAX_VALUE}
-   * bytes.
+   * Validates the user file buffer size is a non-negative number.
    *
-   * @throws IllegalArgumentException if USER_FILE_BUFFER_BYTES bigger than Integer.MAX_VALUE
+   * @throws IllegalStateException if invalid user file buffer size configuration is encountered
```

Comment:
We're moving in the direction of only using unchecked exceptions for issues indicating bugs, so a checked exception or return value would be preferable.